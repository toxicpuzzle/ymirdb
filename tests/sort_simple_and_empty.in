SET a 1
POP a
GET a
SORT a
GET a
SET a 3 2 1
GET a
SORT a
GET a
bye