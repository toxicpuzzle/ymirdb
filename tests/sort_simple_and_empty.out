> ok

> 1

> []

> ok

> []

> ok

> [3 2 1]

> ok

> [1 2 3]

> bye
